static/app/components/replays/timeAndScrubberGrid.tsx

```diff
@@ -58,7 +59,8 @@ export default function TimeAndScrubberGrid({
   isCompact = false,
   showZoom = false,
 }: TimeAndScrubberGridProps) {
-  const {currentTime, replay} = useReplayContext();
+  const {currentTime, replay, timestampType} = useReplayContext();
```

Comment:
I wanna suggest using these lines instead of adding stuff to `useReplayContext`:

```
import {useReplayPrefs} from 'sentry/utils/replays/playback/providers/replayPreferencesContext';

const [prefs] = useReplayPrefs();
const timestampType = prefs.timestamp;

// OR, if you wanna compress it:
const [{timestamp}] = useReplayPrefs();
```

nbd either way, but i have a longer term vision to refactor useReplayContext into some smaller parts